.circleci/config.yml

```diff
@@ -26,6 +26,32 @@ jobs:
       - run:
           name: Build the images
           command: IMAGE_TAG=$(.circleci/scripts/tag.sh) docker buildx bake -f bake.hcl --no-cache
+      - run:
+          name: Scan the images with clair
+          command: |
+             chmod +x ".circleci/scripts/runclairscan.sh"
```

Comment:
The script should have the correct permissions so this isn't necessary.

Reply:
Yeah, I pipeline is failing with the permissions issue if we don't give the permissions.